keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/KeelRepository.kt

```diff
@@ -121,7 +123,7 @@ interface KeelRepository : KeelReadOnlyRepository {
   // START ResourceRepository methods
   fun allResources(callback: (ResourceHeader) -> Unit)
 
-  fun storeResource(resource: Resource<*>)
+  fun storeResource(resource: Resource<*>): Resource<*>
```

Comment:
This method now returns the resource as persisted, i.e. with a `uid` and `version` in the metadata